### PR TITLE
EREGCSC-1553 -- Big contact/sign up calls to action

### PR DESCRIPTION
**Pull request by PhilR8**
Resolves [EREGCSC-1553](https://jiraent.cms.gov/browse/EREGCSC-1553)

**Description**
Add "Contact" and "Sign Up" calls to action at the bottom of the new homepage.

**This pull request changes:**

- Adds `SIGNUP_URL` to AWS Parameter Store and uses it for the "Sign Up For Testing" link
- Edits existing `SURVEY_URL` in AWS Parameter Store to remove `?embedded=true` URL parameter
   - `?embedded=true` is now appended to the end of the URL used for the blocking modal in the `base.html` template
   - `SURVEY_URL` is used for "Contact" link
   - must remember to remove `?embedded=true` from `val` and `prod` param
- Adds Contact/Sign Up For Testing elements and styles them per the [Figma comp](https://www.figma.com/file/Gx6pz0IxWqGsweFVdezKNe/eRegs-Homepage---Multi-View-Update?type=design&node-id=3702-11809)

**Steps to manually verify this change:**

1. visit `homepage_test` on the experimental deployment and ensure that styling matches Figma comp
   - the link color was changed to #EEFAFE and an underline was added to satisfy Axe a11y link style/contrast rules
2. ensure `SIGNUP_URL` is in Parameter Store for all environments

### CODE REVIEW

File: solution/backend/cmcs_regulations/context_processors.py [hunk 1]
```diff
@@ -20,6 +20,12 @@ def survey_url(request):
     }
 
 
+def signup_url(request):
+    return {
+        "SIGNUP_URL": settings.SIGNUP_URL
+    }
+
+
```
Comment (thwalker6) [inline solution/backend/cmcs_regulations/context_processors.py:28]: suprised the linter didnt catch this but one blank line after  a function.  So remove line 27 and also remove line 22.

**Review by thwalker6 — CHANGES_REQUESTED** [review]
small change

File: solution/backend/cmcs_regulations/context_processors.py (hunk 1 above)
Comment (PhilR8) [reply solution/backend/cmcs_regulations/context_processors.py:28]: The linting rule is two lines between classes, not one: https://www.flake8rules.com/rules/E305.html

**Review by thwalker6 — APPROVED** [review]
lgtm

**Pull request merged**